src/content/admin/resource-manager.mdx

```diff
@@ -32,37 +32,21 @@ It calculates the average usage and automatically adjusts the resource requests
 1. Recommendation Generation: The Resource Manager generates new resource requests based on the average usage, applying a correction factor to ensure efficiency
 1. Automatic Application: When a new pod is created, the recommended resource requests are automatically applied unless overridden the annotation `dev.okteto.com/resource-manager: false`
 
-## Enabling and Configuring the Resource Manager
-<!-- ### Enabling the Resource Manager
-The Resource Manager is controlled through the Helm chart configuration. By default, it is enabled with the following settings:
-
-``` yaml
-resourceManager:
-  enabled: true
-  schedule: 300
-  deletePeriod: 15
-  priorityClassName:
-    labels:
-	annotations:
-	resources:
-	recommendations:
-		weight:
-		correction:
-		min:
-			cpu:
-			memory:
-``` -->
+## Configuring the Resource Manager
+### Enabling the Resource Manager
+Resource Manager has been installed in Okteto chart version 1.26 and above. Refer to our [Helm configuration documentation](self-hosted/helm-configuration.mdx#resourcemanager) for available options.
+
 #### To enable the Resource Manager:
 
-<!-- 1. Deploy your Helm chart with the updated configuration -->
+1. Update and deploy your [Helm chart with the Resource Manager configuration](self-hosted/helm-configuration.mdx#resourcemanager) with any changes that pertain to your cluster
 1. Once enabled, the Resource Manager will start a cron job every 5 minutes to compute the recommendations
 
-**Activate the Resource Manager's Recommendations Through the UI:**
+**Activate the Resource Manager's Automatic Recommendations Through the UI:**
 
 1. Log in to the Okteto Admin UI
-1. Navigate to Admin -> General where you can manage system-level settings
-1. Locate the Resource Manager section. If the Resource Manager is installed but not yet enabled, you will see an "Manual/Automatic" toggle switch
-1. Toggle the switch to "Automatic". This will activate the Resource Manager
+1. Navigate to Admin -> Resource Manager
+1. If the Resource Manager is installed but not yet enabled, you will see an "Manual/Automatic" toggle switch
```

Comment:
I would remove "If the Resource Manager is installed but not yet enabled" it's confusing. install == enable in this context. I would use the word "enable" because the Helm value is `resourceManager.enabled`. 

```suggestion
1. If the Resource Manager is enabled, you will see an "Manual/Automatic" toggle switch
```

Reply:
<img width="994" alt="Screenshot 2024-10-31 at 18 34 04" src="https://github.com/user-attachments/assets/57728e9e-430a-488d-9056-51408f5fddff">

for context, when `resourceManager.enabled: false`, the UI shows this CTA linking to https://www.okteto.com/docs/1.26/admin/resource-manager